dnf5/config/usr/share/dnf5/aliases.d/compatibility.conf

```diff
@@ -1,4 +1,4 @@
-version = '1.0'
+qersion = '1.0'
```

Comment:
Oh, this is a typo which needs to be fixed, I guess.

Reply:
Yuck.  I was clearly typing too early yesterday.  Sorry about that.